### ISSUE TITLE
Movie progress indicator (OSD)

### ISSUE THREAD

**Issue by EdwinKM** [phillipberndt/pqiv#165]
A simple on screen progressbar/percentage  would be nice

**Comment by phillipberndt** [comment]
True. Would have to be toggleable I guess. Hardest bit is that backends are not currently passing (or are even aware of) the length of animated images, so that'd have to be added.
